### ISSUE TITLE
external_network_gateway not filled in datasource vcd_network_direct

### ISSUE THREAD

**Issue by NilsBusche** [terraform-providers/terraform-provider-vcd#450]
### Terraform Version
Terraform v0.12.18

### Affected Resource(s)
`vcd_network_direct`

### Terraform Configuration Files
```hcl
data "vcd_network_direct" "tf-network-direct" {
  name = "EXT_BACKUP01"
}

output "test" {
  value = data.vcd_network_direct.tf-network-direct
}
```

### Expected Behavior
`external_network_gateway` should be set to the gateway of the external network

### Actual Behavior
`external_network_gateway` is not filled:

terraform.tfstate:

```
"external_network_gateway": null,
```

### Steps to Reproduce
`terraform apply`

**Comment by NilsBusche** [comment]
After a detailed look into the code I found out this issue is caused by missing statement in func `genericVcdNetworkDirectRead`.

Adding `_ = d.Set("external_network_gateway", currentNetwork.DefaultGateway)` after the following line solves the issue: https://github.com/terraform-providers/terraform-provider-vcd/blob/e8206042c21830f63d1f4abadc70a6461402225a/vcd/resource_vcd_network_direct.go#L193

Should I create a Pull Request for fixing this?

**Comment by dataclouder** [comment]
Thanks for reporting the issue.

> Should I create a Pull Request for fixing this?

No need. I will fix it in my next PR. This addition needs to have a test, and I can add it more easily than others.

**Comment by lvirbalas** [comment]
@NilsBusche thank you very much for identifying this so clearly and finding the cause!

**Comment by NilsBusche** [comment]
@dataclouder Is there already an update for this issue?

**Comment by dataclouder** [comment]
There is a change ready, but it didn't make the cut for the current release (PR #465)

**Comment by dataclouder** [comment]
Fixed in PR #465
